components/gfx/display_list/mod.rs

```diff
@@ -1043,6 +1043,22 @@ impl ClippingRegion {
     }
 }
 
+impl fmt::Debug for ClippingRegion {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if self.main == max_rect() && self.complex.is_empty() {
+            write!(f, "ClippingRegion::Max")
+        } else if self.main == max_rect() && self.complex.is_empty() {
```

Comment:
Though I wonder if you really want self.main == Rect::zero() here, since this branch is otherwise dead code.